### DISCUSSION TITLE
Seed data in migration without using ActiveModel?

### DISCUSSION THREAD

**Discussion by gd87429** [SeaQL/sea-orm#1058] [Q&A] [answered]
In https://www.sea-ql.org/SeaORM/docs/migration/seeding-data/ the insert statement is using `cake::ActiveModel`. But wouldn't that break functionality if the following happens:

1. Migration `#1` create a table with column `test_column`
2. Migraiton `#2` seeds data using `ActiveModel { test_column: whatever }`
3. Migration `#3` deletes column `test_column`
4. At this point `ActiveModel` does not contain `test_column`

Now when a **new** user runs all migrations for the first time, wouldn't migration `#2` break because up-to-date `ActiveModel` does not have column `test_column` anymore and therefore can't be referenced?

Is there a way to seed with data without using `ActiveModel` **and** without using raw SQL so it still works across all 3 db types?

**Comment by billy1624** [comment, marked as answer]
Hey @gd87429, welcome!! You points are valid. It's doable and easy to run migration script written in SeaQuery. See here for demo
- https://github.com/SeaQL/sea-orm/pull/1063

I will also update the docs to mention th "how to"

**Reply by billy1624** [reply]
Docs updated: https://www.sea-ql.org/SeaORM/docs/migration/seeding-data/

**Comment by gd87429** [comment]
@billy1624 Thank you for taking the time 🙏 This is exactly what I was looking for. Works really well.
